human_aware_rl/imitation/behavior_cloning_tf2_test.py

```diff
@@ -58,8 +58,7 @@ def test_model_construction(self):
             self.assertTrue(np.allclose(model(self.dummy_input), self.expected["test_model_construction"]))
 
     def test_save_and_load(self):
-        model = build_bc_model(**self.bc_params)
-        save_bc_model(self.model_dir, model, self.bc_params)
+        model = train_bc_model(self.model_dir, self.bc_params)
```

Comment:
Is there a reason why you changed this? My initial idea was the unit tests are more modular if this one doesn't rely on the "training" portion of the BC code, as in it only saves and loads

Reply:
Sounds good.